### PR TITLE
(docs): update envoy plugin docs to match new configurable grpc msg sizes

### PR DESCRIPTION
**Pull request by emaincourt**
Signed-off-by: Elliot Maincourt <e.maincourt@gmail.com>

Hi,

This PR updates documentation for `opa-envoy-plugin` to match this [change](https://github.com/open-policy-agent/opa-envoy-plugin/pull/323) that allows to configure GRPC server's max rcv/send message sizes.

Thanks in advance for reviewing

### CODE REVIEW

File: docs/content/envoy-introduction.md
Comment (srenatus) [inline docs/content/envoy-introduction.md:53]: [nit] I'm not sure everyone knows roughly what `math.MaxInt32` means in this context. Can we spell it out, or give some other indication, please?

**Review by srenatus — COMMENTED** [review]
thanks for adding this. gave it a quick review, some notes 👇

File: docs/content/envoy-introduction.md
Comment (emaincourt) [reply docs/content/envoy-introduction.md]: Sorry, I should have caught this one.

File: docs/content/envoy-introduction.md
Comment (emaincourt) [reply docs/content/envoy-introduction.md]: Sure, I've replaced it with a more explicit description.

**Comment by emaincourt** [comment]
@srenatus Thanks a lot for your review. I should have addressed all the comments. Does it look better to you now ?

**Review by srenatus — DISMISSED** [review]
Thanks. I take it we'll wait for the opa-envoy-plugin PR to merge?

**Comment by emaincourt** [comment]
Thanks ! Yes, hopefully my last changes will match the requirement. Once merged I’ll change the draft state of this PR.

**Comment by emaincourt** [comment]
@srenatus The [pull request](https://github.com/open-policy-agent/opa-envoy-plugin/pull/323) was merged. I rebased this branch and squashed all the commits. I let you see if it still looks good to you. Thanks.

**Review by srenatus — APPROVED** [review]

**Pull request merged**
